### ISSUE TITLE
Missing template field on newly created diagram

### ISSUE THREAD

**Issue by szerie** [camunda/camunda-modeler#2017]
__Describe the Bug__

On Camunda Modeler 4.2.0 it is required to create new diagram, save it and then open application once again to have template field available. Otherwise applying template on any task is not possible.

__Steps to Reproduce__

1. Open Camunda Modeler 4.2.0.
2. Create task (applicable templates exists).
3. Select newly created task - template field is missing.

![missingTemplateField1](https://user-images.githubusercontent.com/57395064/99071696-ed441300-25b2-11eb-8e3c-2189a61efe78.PNG)

4. Save diagram.
5. Close Camunda Modeler.
6. Open Camunda Modeler once again and load previously saved diagram. Template field is visible:
![missingTemplateField2](https://user-images.githubusercontent.com/57395064/99071758-064cc400-25b3-11eb-8030-b4b8762c6a4e.PNG)

__Expected Behavior__

Template field should be visible from the beginning. Saving diagram and restarting Camunda Modeler application should not be required.

__Environment__

- OS: Windows 10
- Camunda Modeler Version: 4.2.0

**Comment by nikku** [comment]
Can you reproduce the same issue in the current (v4.4.0) version of the app?

**Comment by szerie** [comment]
> Can you reproduce the same issue in the current (v4.4.0) version of the app?

No, it is not reproducible on 4.4.0, but due to [Issue 2011](https://github.com/camunda/camunda-modeler/issues/2011) I can't use newer version at the moment.

**Comment by nikku** [comment]
What is the last good known version this worked for you?

**Comment by philippfromme** [comment]
Where are you storing your element templates? I can only reproduce this with local element templates stored in a `.camunda` directory which is expected as new diagrams are not saved anywhere on disc.

**Comment by szerie** [comment]
> What is the last good known version this worked for you?

I did not checked. I just started using Camunda Modeler in version 4.2.0 and then I tried to use 4.4.0 when I noticed that it is available.

>Where are you storing your element templates?
Yes, I am using local element templates stored in a .camunda directory. What are different options? In documentation I found only nfo about .camunda and resources\element-templates directories. Did I miss sth?

**Comment by nikku** [comment]
The options are [clearly outlined in our documentation](https://github.com/camunda/camunda-modeler/tree/develop/docs/element-templates#configuring-templates).

**Comment by philippfromme** [comment]
@szerie Local templates that are stored in `.camunda` directories are not found in v4.3.0 and v4.4.0 due to a [bug](https://github.com/camunda/camunda-modeler/issues/2012). This issue has been fixed so you can expect it to be working again in the next release. Sorry for the inconvenience.

**Comment by nikku** [comment]
Thanks for your input / feedback regarding this.

I've updated the issue description and moved the issue to the backlog.

__TLDR:__ When saving a diagram so that local `.camunda` stored templates are available the chooser should be shown right after save. This is a long standing bug. 

To workaround you may reload the app via `CTRL/CMD+R`.
